lib/foreman_discovery/engine.rb

```diff
@@ -107,7 +107,10 @@ class Engine < ::Rails::Engine
         allowed_template_helpers :rand, :facts_hash
 
         # apipie API documentation
-        apipie_documented_controllers ["#{ForemanDiscovery::Engine.root}/app/controllers/api/v2/*.rb"]
+        # Only available in 1.8, otherwise it has to be in the initializer below
+        if (SETTINGS[:version].to_s.include?('develop') or Gem::Version.new(SETTINGS[:version]) >= Gem::Version.new('1.8'))
```

Comment:
I am fine with this. Filed issue ticket for 2.1 to remove this workaround: http://projects.theforeman.org/issues/9017

Going to test this. Thanks.